accounting/accounting_cli_functions.py

```diff
@@ -248,6 +248,74 @@ def view_jobs_before_end_time(conn, time_before, output_file):
     return job_records
 
 
+def add_bank(conn, bank, shares, parent_bank=""):
+    parent_bank_shares = 0
+    total_sub_bank_shares = 0
+
+    # if the parent bank is not "", that means the account
+    # trying to be added wants to be placed under a parent bank
+    if parent_bank != "":
```

Comment:
Can the bank hierarchy share invariant/inequality checks be factored out so that you can use the same methods for either `add` or `edit`? If a bug is introduced, it is always to fix one place as opposed to multiple duplicate places.